app.py

```diff
@@ -134,6 +136,24 @@ async def command_restart(message, args):
     else:
       await message.channel.send(f"{vm['name']} is not currently running.")
 
+async def command_stop(message, args):
+    if len(args) < 1 or args[0] not in vms.keys():
+      await message.reply(f'Must specify one of: {configured_vms_str}')
+      return
+
+    vm = vms[args[0]]
+    status = vm_power_state(vm)
+    if status == 'starting':
+        await message.reply(f"{vm['name']} is just starting")
+    elif status =='deallocated' or status == 'stopped':
```

Comment:
```suggestion
    elif status == 'stopped':
```
I think we only need to check if status is `stopped` here. `vm_power_state()` will return `stopped` if the actual state is `deallocated`.

Reply:
That makes sense - updating and also removing the extra `elif` for checking the other two stopping-type statuses.